### ISSUE TITLE
Global Fire 

### ISSUE THREAD

**Issue by Snekili** [datasets/registry#145]

**Comment by Snekili** [comment]
Data > http://www.fire.uni-freiburg.de/current/globalfire.htm
http://neo.sci.gsfc.nasa.gov/view.php?datasetId=MOD14A1_M_FIRE

**Comment by zelima** [comment]
@Snekili @rgrp to download data it need first to be filtered. 
EM-DAT may be filtered by year, disaster type or by it's subgroup. eg: We can get data that contains total damage (USD) by countries that was caused by earthquake. It looks something like this:
```
Country,Type,Date,Total damage ('000 US$)
China,Earthquake,27-07-1976,242000
Haiti,Earthquake,12-01-2010,222570
China,Earthquake,22-05-1927,200000
China,Earthquake,16-12-1920,180000
Indonesia,Earthquake,26-12-2004,165708
```
Problem is that the button that gives download link is JS button (I guess), so I can not get actual download link to automate process.

It also has [disaster-list](http://www.emdat.be/disaster_list/index.html) where you can request raw data, but first we need to request data. So I'll have to manually download filtered file and datapackage after

Other link that contains active fires data also needs to be filtered, and gives the links to download files, but when I do so the data looks very strange eg:
if i choose data of April 2016 (1 degree) the first couple of lines look something like this: 
```
99999.0,99999.0,99999.0,99999.0,99999.0,99999.0,
99999.0,99999.0,99999.0,99999.0,99999.0,99999.0,
99999.0,99999.0,99999.0,99999.0,99999.0,99999.0,
99999.0,99999.0,99999.0,99999.0,99999.0,99999.0,
99999.0,99999.0,99999.0,99999.0,99999.0,99999.0,
99999.0,99999.0,99999.0,99999.0,99999.0,99999.0,
```
To be honest all the data looks like this - see link: http://neo.sci.gsfc.nasa.gov/servlet/RenderData?si=1703042&cs=rgb&format=CSV&width=360&height=180

**Comment by rgrp** [comment]
@Snekili I am going to let you look at this with @zelima

**Comment by Snekili** [comment]
@rgrp 
@zelima You have example in google drive link  - [**Natural__1900_2015_summary.csv - Climatilogical**](https://drive.google.com/open?id=1pQjk6k6zos6uo-x9QCxBGHSmxUu5mpHpSr-YGDGspWc), but  this database contains much more than just a Global Fire and damage, It has datasets for Drought, Earthquakes, Epidemic, Extreme Temperature, Volcanic Activity etc... and more advanced subgroups like Complex disasters, Natural, Technological... Climatological, Meteorological..... By Continent, Region or Country... 
And I cant get link to download csv file directly, I can just download it in drive. 
We must look at it together.

**Comment by zelima** [comment]
@Snekili Yeah, there are much of useful data, just there are no links to download them. 
@rgrp there is kind of "online tool" that generates the data depended on filter options. You can try it here: http://www.emdat.be/disaster_profiles/index.html
